js/components/operatorSignIn/attestation.tsx

```diff
@@ -47,22 +51,37 @@ export const Attestation = ({
     <div className="text-sm">
       Step 2 of 2
       <SignInText />
-      <SignaturePrompt defaultValue={defaultValue} onChange={setEntered} />
-      <SignatureHint badge={badge} signatureText={entered} />
-      <p className="my-3">
-        By pressing the button below I, <b className="fs-mask">{name}</b>,
-        confirm the above is true.
-      </p>
-      <button
-        className={className([
-          "block w-full md:max-w-64 mx-auto h-10 px-5 bg-gray-500 text-gray-200 rounded-md",
-          (!ready || loading) && "opacity-50",
-        ])}
-        onClick={onComplete}
-        disabled={!ready}
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+        }}
       >
-        Complete Fit for Duty Check
-      </button>
+        <InputBox
+          title={"Operator Badge Number"}
+          defaultValue={defaultValue}
+          onChange={setEnteredBadge}
+        />
+        <SignatureHint badge={badge} signatureText={enteredBadge} />
+        <InputBox
```

Comment:
Do you still want the `removeLeadingZero` behavior in the radio input?

Reply:
This mattered less when radio number was a string; fixed now. thanks! 0f77073